pages/docs/specifications/2.0.0.md

```diff
@@ -1,8 +1,4 @@
----
-title: "AsyncAPI specification 2.0.0"
-weight: 1
----
-
+# AsyncAPI Specification
 #### Disclaimer
```

Comment:
make sure it reflects 1:1 https://github.com/asyncapi/spec/blob/v2.0.0/versions/2.0.0/asyncapi.md